scripts/docker-push.sh

```diff
@@ -52,10 +52,16 @@ tag_and_push ()
 # Possible docker image tags
 IMAGE_TAG_EDGE="edge-php${VERSION}"
 IMAGE_TAG_STABLE="php${VERSION}"
+
+IFS='.'
```

Comment:
Put `IFS='.'` on the same lien with `read -a ...`, so it only applies to that line.

Reply:
done